packages/athena/sodasql/dialects/athena_dialect.py

```diff
@@ -72,6 +74,23 @@ def create_connection(self):
             catalog_name=self.catalog)
         return conn
 
+    def __query_table(self, table_name):
```

Comment:
Looks like this can go into the  Dialect class, and we don't need to repeat it in every dialect.